src/pages/imprint-page.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <v-container>
+    <router-link to="/">
```

Comment:
```suggestion
    <router-link :to="{name: 'index'}">
```

Immer named routes verwenden ;)

Reply:
ok